Moq.AutoMock.Tests/ResolvesEnumerables.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq.AutoMock.Resolvers;
+
+namespace Moq.AutoMock.Tests
+{
+    [TestClass]
+    public class ResolvesEnumerables
+    {
+        [TestMethod]
+        public void ResolvesObjectEnumerableFromContainer() => Resolves(new object());
+
+        [TestMethod]
+        public void ResolvesNumberEnumerableFromContainer() => Resolves(42L);
+
+        [TestMethod]
+        public void ResolvesStringEnumerableFromContainer() => Resolves(nameof(ResolvesEnumerables));
+
+        private static void Resolves<T>(T expected)
+        {
+            var mocker = new AutoMocker { Resolvers = { new EnumerableResolver() } };
```

Comment:
The test would be a bit more robust if we didn't set `Resolvers`

Reply:
That just copied from the LazyResolver. I figured there's a good point in setting the resolvers to the single resolver only.

Reply:
I'm not sure why the setter is done in the lazy tests, the same logic would apply for robustness and tests pass without it there too. Probably good to remove both. (Not that you need to remove the Lazy one)

Reply:
Done.